life/modernized/02-chapter-1.adoc

```diff
@@ -1,7 +1,8 @@
 == Chapter I.
 
-A Short Sketch of Francis Howgill`'s Life;
-with an Extract from His Own Account of His Early Religious Experience.
+[.chapter-synopsis]
```

Comment:
This was one sentence before, and I initially put it as a [.section-summary-preface], but then going on to chapter two, it had a chapter synopsis, and it looks that way in further chapters too so I broke it  up and made it a CS also.

Reply:
That looks good to me.